### DISCUSSION TITLE
Hover effect

### DISCUSSION THREAD

**Discussion by Nestorpf1124** [tailwindlabs/tailwindcss#2127] [Help]
Hi, I'm looking for a hover effect like this one overflow.io on the "Price" tag, I've been trying for a while but I can't get it. Thanks for your help.

**Comment by ypk** [comment]
Hi, I know it's a bit late, but are you asking about the effect in the site menu?

**Reply by Nestorpf1124** [reply]
Hi, yeah, the hover effect on the site menu 😊.

**Reply by ypk** [reply]
Hi, I tried my best to recreate the effect. I achieved results somewhat close to the one on the site.

I've commited the code to codesandbox here: https://codesandbox.io/s/tailwind-menu-hover-effect-5osvs

There's 2 solutions, one with JavaScript and one with pure css.
